### PR TITLE
Update and overhaul survivor masks

### PR DESCRIPTION
**Pull request by worm-girl**
#### Summary
Bugfixes "Update and overhaul survivor masks"

#### Purpose of change

#69555 moved us to a new model for resisting fluid attacks. I want to use this model in the future to improve how the game handles acid, but I discovered that the survivor mask items are badly outdated and not really realistic in any way. As with a lot of the old survivor stuff, it was just magically slapping Kevlar on an item to make it better somehow. This made for a highly breathable mask, which is not something you want from your PPE.

Survivor gear is a major part of the game's identity, but it ought to make sense and fill a niche that other gear doesn't. People's expectations are that the gear should incorporate simple, common(ish) materials, solid construction, mobility, and be specifically tailored to deal with the Cataclysm in a way that, for instance, an ESAPI vest or a steel great helm are not.

#### Describe the solution

- Survivor masks are now described as being made of treated leather, similar to a lederschutzmasken ( https://collections.tepapa.govt.nz/object/1345578 ). Rubber is superior to leather for a gas mask, but the player doesn't currently have access to the right kind of rubber molding, and if they did, they'd probably just make a regular gas mask. Treated leather is airtight enough for things like tear gas, smoke and some forms of poison gas - I assume our poison gas is mostly hydrogen sulfide, as it appears naturally in caves and is produced by rotting corpses
- Treated leather is a new material that is like leather but has 0 breathability and one more acid resistance. Technically any sufficiently thick chrome-tanned leather can be impermeable, but we only have one type of tanning currently. Work boots and boiled leather armor should also be made of this stuff, or at least mostly made of it
- Winter survivor masks have been obsoleted. There's nowhere to put fur on a gas mask that would make it warmer, and if this even worked you'd see people using them in extremely cold conditions IRL
- The light survivor mask is now a half-mask, with no goggles. The heavy survivor mask has a steel faceplate stuck onto it
- The lenses are all made of transparent impact resistant plastic. This is what ballistic glasses are (supposed to be) made of, and was a totally unused material sitting in our files waiting for someone to update old plastic armor. No more using soda bottles for lenses on your high-performance MacGyver gear
- I updated goggles, the rigid plastic sheet, and the full-face motorcycle helmet to use this material. I also made chunks of it. Polycarbonate is the go-to for any kind of visor that needs to stand up to impact or shrapnel, we can probably use it in a lot of places
- All survivor masks will now fit on people who have muzzles. This is done by adding a new flag, UNRESTRICTED, that ignores mutation restrictions in the same way OVERSIZE does. OVERSIZE still works for now, but will need to be redone at some point. OVERSIZE was initially created back when we didn't have any tiny mutants, and problematically makes an item both XL and mutant-appropriate. This meant that tiny mutants with muzzles (IE mouse) had no way of wearing a gas mask at all. Having UNRESTRICTED in place will allow us to decouple the two concepts and better differentiate big things versus things that mutants can wear
- Our encumbrance values for gas masks are all over the place, with some being prohibitively high or way too low for no reason I can discern. They all use the same or roughly analagous filters, and should all be restricting breathing in the same way. I adjusted all gas masks to impose 25 mouth encumbrance when fitted, with minor penalties for the armored ones

#### Describe alternatives you've considered

Our current gas system is quite bad, with enviro offering full protection only after specific breakpoints. If gear doesn't hit the breakpoint, it's effectively useless. I would like these to be less reliable than factory-made masks, but we'd need to make the system more granular first.

A PAPR system, which creates positive pressure in a gas mask and greatly alleviates what our game calls mouth encumbrance, would be a great idea. You would find them on both soldiers and firefighters. It'd just be a battery-powered thing that made your life easier while it was on.

#### Testing

Spawned all masks, saw they had the appropriate protection and encumbrance values. Checked the crafting recipes for any wonkiness.

#### Additional context

image

### CODE REVIEW

File: data/json/items/tool_armor.json [hunk 1]
```diff
@@ -2026,23 +2026,12 @@
     "type": "TOOL_ARMOR",
     "category": "armor",
     "name": { "str": "survivor firemask" },
-    "description": "A custom-built, Nomex-insulated gas mask that covers the face and eyes.  It provides excellent protection from heat, smoke, tear gas, and shrapnel.  It must be prepared before use.",
-    "weight": "708 g",
+    "description": "A custom-made leather gasmask faced with nomex.  It must be prepared before use.",
```
Comment (TheShadowFerret) [inline data/json/items/tool_armor.json:2029]:
```suggestion
    "description": "A custom-made leather gasmask faced with Nomex.  It must be prepared before use.",
```
*Angry DuPont noises*

**Comment by fairyarmadillo** [comment]
@TheShadowFerret I forget every time 😆

**Comment by OtpalTS** [comment]
Why make the steel faceplate out of low carbon steel? Why not case hardened or even tempered for the most protection?

**Comment by fairyarmadillo** [comment]
> Why make the steel faceplate out of low carbon steel? Why not case hardened or even tempered for the most protection?

Because Survivor gear is homemade out of easy-to-acquire (in the apocalypse) materials. It's also meant to be easily repaired on the go. You rarely ever get hit in the face, and a point or two of damage would not be worth the tradeoff.

Using some crazy level of steel would make crafting and repairing it a huge pain, far more than a mask made of oiled leather is worth. If you want crazy tempered high-tech stuff, go with Nomad gear.

**Comment by IdleSol** [comment]
I apologize for the quality of the text. I used the Deepl translator.

I realize I'm late with my comments.

1. Something I'm confused about. To create a heavy mask, you need "sheet_metal_small", ie steel. But the result is low carbon steel. Why?

2. What is the eye encumbrance based on? Is this plastic so murky that you can't see anything through it? Just for comparison, sunglasses, now have 0 encumbrance. Perhaps this is a flaw in the sunglasses.
What is encumbrance in relation to the eyes? Transparency of the glass? Comfort?

3. Have you considered sequential creation? Heavy mask = survivor mask + small metal sheet = half mask + survivor goggles + small metal sheet.

4. I propose to increase the capacity of the mask to 2 cartridges, to be able to fully utilize the resource.

5. In my dreams, my survivor would wear an advanced mask. Something like: mask + Respirator CBM + Anti-Glare Compensators CBM + Implanted Night Vision CBM. Naturally with 9-10 level in tailoring, computer, fabrication and electronics skills.

**Comment by fairyarmadillo** [comment]
> 1. Something I'm confused about. To create a heavy mask, you need "sheet_metal_small", ie steel. But the result is low carbon steel. Why?

Items made of sheet metal are usually low carbon steel, such as this: https://cdda-guide.nornagon.net/item/helmet_metal_sheets

> 2. What is the eye encumbrance based on? Is this plastic so murky that you can't see anything through it? Just for comparison, sunglasses, now have 0 encumbrance. Perhaps this is a flaw in the sunglasses.

These are not sunglasses, they are an airtight visor. Ski goggles, a similar item, have 15 encumbrance. The heavy survivor mask has metal protection which further reduces vision.

image

The mask is something like this. It would be hard to see out of that.

> 3. Have you considered sequential creation? Heavy mask = survivor mask + small metal sheet = half mask + survivor goggles + small metal sheet.

That is a good idea.

> 4. I propose to increase the capacity of the mask to 2 cartridges, to be able to fully utilize the resource.

No other mask works this way. I am not sure if it would be possible.

> 5. In my dreams, my survivor would wear an advanced mask. Something like: mask + Respirator CBM + Anti-Glare Compensators CBM + Implanted Night Vision CBM. Naturally with 9-10 level in tailoring, computer, fabrication and electronics skills.

Survivor gear is low-tech and hand-made. For advanced gear, you should look for something like this: https://cdda-guide.nornagon.net/item/power_armor_helmet_basic

**Comment by IdleSol** [comment]
>Items made of sheet metal are usually low carbon steel, such as this: https://cdda-guide.nornagon.net/item/helmet_metal_sheets

A helmet for a mask?
https://cdda-guide.nornagon.net/item/sheet_metal_small

>These are not sunglasses, they are an airtight visor. Ski goggles, a similar item, have 15 encumbrance. 
Survivor's Mask includes: 
- pair of ballistic glasses (encumbrance = 5); https://cdda-guide.nornagon.net/item/glasses_bal
- survivor goggles (encumbrance = 5, encumbrance of components varies from 0 (sunglasses) to 40); https://cdda-guide.nornagon.net/item/survivor_goggles
- rigid plastic sheet.

5 or 5 and the result was 15. 

Sunglasses, this is more as an example. The glasses in them are quite dark, but the encumbrance is quite low.
pair of fit-over sunglasses, encumbrance = 0 https://cdda-guide.nornagon.net/item/fitover_sunglasses
pair of stylish sunglasses, encumbrance = 1 https://cdda-guide.nornagon.net/item/fancy_sunglasses

But I do agree that using regular goggles for protection, not a good idea.

>The mask is something like this. It would be hard to see out of that.

But we're using more modern technology. The new plastic will allow us to create something like this:
m50-us-military-gas-mask
https://www.avon-protection.com/products/m50/5223

Given the use of ballistic goggles, something along the lines of this.
fm51-combat-vehicle-cbrn-gas-mask
https://www.avon-protection.com/products/fm51/5226
Just replace the plastic with leather? And the glass is clear.

While looking for examples, I came across an interesting option for an alternate version of the heavy mask
avon-protection_hmk150-riot-helmet-gas-mask_header
And it's a hinged plastic visor. Which gives you 100% coverage of your entire face. Just note that in the picture it's part of the helmet.

In fact, the longer I think about it, the more I like this version of the survivor's mask.

Respirator for respiratory protection. Goggles to protect eyes from dust with priority on reducing encumbrance. Aim to see in the dark as well as without them. And the top is covered by a transparent visor. With emphasis on protective features including acid and glare protection, i.e. can be blacked out.

And then, during the day and in the light, the visor is lowered. If it is necessary to go down to a dark place and not to shine, the visor is raised. Yes, the protection is less, but the stealth is higher. And from the impact of gas or mucus on the eyes, will protect the goggles.

A nice addition, the mask itself can be less burdensome and with less warmth.

>Survivor gear is low-tech and hand-made. For advanced gear, you should look for something like this

That said, the survivor does a lot of much more complicated things.

Off-topic. The problem is that with certain play styles, you can't access a lot of off-the-shelf and advanced stuff. As a simple example, the multiplier on stuff is x0.1.

And the game is for the champion of purity: no bionics, no mutation, only 100% human. Turns into another problem. Add a faction limit and the best outfit option is a survivor outfit or steel armor.

**Pull request merged**